### PR TITLE
Adding additional context for deblur tables

### PR DESCRIPTION
**Pull request by adswafford**

### CODE REVIEW

File: docs/qiita-16S-analysis.rst [hunk 1]
```diff
@@ -430,6 +430,10 @@ First, because the deblur process creates two separate BIOM tables, you’ll wan
 
 The specific ID for your table will be unique, so make a note of it, and you can use it to select the correct table for analysis.
 
+Qiita employs Deblur v1.0.4 on trimmed sequences (UC San Diego studies are typically processed with 90, 100, and 150 nt) with no minimum read threshold count, followed by insertion into the latest (Aug 2013) 99% OTU tree from GreenGenes using SEPP as part of the Qiime2 q2-fragment-insertion plugin.
```
Comment (tkosciolek) [inline docs/qiita-16S-analysis.rst]: should read:
```
Qiita employs Deblur v1.0.4 on trimmed sequences (UC San Diego studies are typically processed with 90, 100, and 150 nt) with no minimum read threshold count, followed by insertion into the latest (Aug 2013) 99% OTU tree from GreenGenes using SEPP as part of the QIIME2 `q2-fragment-insertion plugin<https://github.com/biocore/q2-fragment-insertion>`__.
```

File: docs/qiita-16S-analysis.rst [hunk 2]
```diff
@@ -430,6 +430,10 @@ First, because the deblur process creates two separate BIOM tables, you’ll wan
...
+
+In addition, by deblurred sequences in Qiita use a reduced threshold for sequence abundance across all samples of 1 read rather than the 10 read threshold default in deblur. The single read threshold was chosen (rather than the default 10 read threshold) to support the best practice of setting a custom minimum read threshold that is suitable to your single-study analysis or meta-analysis. This can be done using the Filter Table command (see below).
```
Comment (tkosciolek) [inline docs/qiita-16S-analysis.rst]: Filter Table command in the last sentence should include a link

**Review by tkosciolek — CHANGES_REQUESTED** [review]

**Comment by stephanieorch** [comment]
@tkosciol hyperlink is fixed and I believe I set up the anchor the correct way so everything should be working fine now

**Comment by tkosciolek** [comment]
@stephanieorch in preview the anchor still does not look correct

**Comment by stephanieorch** [comment]
@tkosciol  This is how I did it in the qiita help documentation. It doesn't look correct until it's pushed through. So it should work correctly.

**Pull request merged**